src/main.cpp

```diff
@@ -4727,10 +4727,11 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             RelayTransaction(tx);
             vWorkQueue.push_back(inv.hash);
 
-            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u kB)\n",
+            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u MiB, cache=%.1fMiB(%utx))\n",
```

Comment:
Unfortunately, the other place that logs the cache size is MiB as well: https://github.com/bitcoin/bitcoin/blob/master/src/main.cpp#L2319

-1 on changing the mempool size to be in MiB as well though - note that fees are per 1000, not per 1024, so apart from SI standards adherence kB/MB makes more sense in the context.